v3/src/data-interactive/data-interactive-data-set-types.ts

```diff
@@ -114,3 +114,5 @@ export interface DIUpdateItemResult {
 export interface DIDeleteCollectionResult {
   collections?: number[]
 }
+
+export type DICategoryColorMap = Record<string, string | undefined>
```

Comment:
Does including `undefined` as a possible value make this type more useful? It seems to complicate the actual handler code, and I suspect it's not necessary.